packages/client/src/pages/PortalSettings/categories/integration/SingleSignOn/sub-components/AddSpCertificateModal.js

```diff
@@ -51,6 +51,7 @@ const AddSpCertificateModal = (props) => {
         </Text>
 
         <TextArea
+          id="spCertificate"
```

Comment:
Rename "spCertificate" to "sp-certificate"

Reply:
Fix to "sp-certificate"